src/types/package.rs

```diff
@@ -567,7 +567,8 @@ pub struct PackageStatus {
     pub num_dependencies: u32,
     /// Number of vulnerabilities found in this package and all transitive
     /// dependencies
-    pub num_vulnerabilities: u32,
+    #[serde(default)]
+    pub num_vulnerabilities: Option<u32>,
```

Comment:
Technically `u32` would also work with `#[serde(default)]`, but I prefer making this optional for visibility and to ensure it's a breaking change. That way we will definitely notice if we didn't think about some place where it might be used.

I considered just renaming it to `_num_vulnerabilities` with a `#[serde(rename = ...)]`, but I think having it as a proper `Option` nicely represents the intention that this field might not always be present.

Reply:
`#[serde(default)]` doesn't actually do anything here since `Option<_>` already deserializes as `None` if the field is missing.